cli/src/cmd/init.js

```diff
@@ -23,48 +22,42 @@ export async function init() {
         })
       )
     );
-    const continueAnswer = await spinner.prompt({
-      type: 'confirm',
-      name: 'continue',
+
+    const { projectName } = await spinner.prompt({
+      type: 'text',
+      name: 'projectName',
       message:
-        'A new project will be created in the current directory. Do you want to continue?',
+        "What's your project name? (A folder with this name will be created)",
+      initial: targetDir,
     });
-    if (!continueAnswer.continue) {
-      process.exit(1);
-    }
 
-    let folderCreated;
-    if (!(await checkIfFolderIsClean())) {
-      const folderName = 'hello-world';
-      const { createHelloWorldFolder } = await spinner.prompt({
-        type: 'confirm',
-        name: 'createHelloWorldFolder',
-        message: `Want to run "mkdir ${folderName} && cd ${folderName}"?`,
-      });
-      if (!createHelloWorldFolder) {
-        process.exit(1);
-      }
-      folderCreated = await createProjectFolder(folderName);
+    if (await folderExists(projectName)) {
+      spinner.fail(
+        `Target directory "${targetDir}" already exists. Remove it or choose a different name.`
+      );
+      process.exit(1);
     }
 
-    const currentFolderName = process.cwd().split('/').pop();
-    const { projectName } = await spinner.prompt({
-      type: 'input',
-      name: 'projectName',
-      message: 'What is the name of your project?',
-      default: currentFolderName,
-    });
+    await createProjectFolder(projectName);
 
+    let packageName;
     if (!isValidPackageName(projectName)) {
-      spinner.fail('Invalid package.json name');
-      process.exit(1);
+      const packageNameAnswer = await spinner.prompt({
+        type: 'text',
+        name: 'packageName',
+        message: 'Package name:',
+        initial: targetDir,
+      });
+      packageName = packageNameAnswer.packageName;
+    } else {
+      packageName = projectName;
     }
```

Comment:
is it necessary ? I think in many framework as hardhat, react, vueJS the package name is define by them. Moreover is not very important for the builder because it will not be deployed on NPM. May be you can set "iExec iapp"

Reply:
Good point, maybe we can simplify and remove this optional question. I'll put a hard-coded value like you mentioned and it is quite easy to change anyway.